ReactiveLists.podspec

```diff
@@ -0,0 +1,18 @@
+Pod::Spec.new do |s|
+  s.name             = "ReactiveLists"
+  s.version          = "0.0.1"
+  s.summary          = "React-like API for UITableView and UICollectionView"
+  s.description      = <<-DESC
+                        React-like API for UITableView and UICollectionView 
+                        DESC
```

Comment:
never really got this syntax.

can't this also just be `s.description = "React-like API for UITableView and UICollectionView"` ?

Reply:
Syntax for multi-line. But with brief description agree that plain string is better.